prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -51,16 +71,15 @@ def smart_charging(
     # load NHTS data from function
     if veh_type.lower() == "ldv":
         newdata = data_helper.remove_ldt(data_helper.load_data(census_region, filepath))
+        # updates the weekend and weekday values in the nhts data
+        newdata = data_helper.update_if_weekend(newdata)
```

Comment:
Do we need this for `ldt` as well?

Reply:
Yes we do

Reply:
@dmuldrew Given this missing logic, the integration test still passes. Shall we update the test case to include at least one weekend to cover this case?

Reply:
Maybe, I think the issue here is we don't have a test case for `ldt`...

Reply:
Should we add it to the `ldt` branch below then?